#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("joint1", double_t, 0, "Joint 1 Velocity", 0.0, -2.0, 2.0)
gen.add("joint2", double_t, 0, "Joint 2 Velocity", 0.0, -2.0, 2.0)
gen.add("joint3", double_t, 0, "Joint 3 Velocity", 0.0, -2.0, 2.0)

exit(gen.generate("smm_screws", "smm_screws", "JointVelocities"))
